03-Node-npm-modules/artist.js

```diff
@@ -0,0 +1,19 @@
+var Artist = function(name, awards){
+	this.name = name;
+	this.awards = awards || [];
+};
+
+Artist.prototype.recognice = function(){
+	var msg = this.name + " has the following awards: ";
+	
+	for(var i = 0; i < this.awards.length; i++){
+		msg += this.awards[i] + ' ';
+	};
+    return msg;
```

Comment:
use `message` instead of `msg`, more characters are not an issue and message is more detailed than msg